src/main/kotlin/model/Delius.kt

```diff
@@ -44,6 +44,7 @@ class Delius private constructor() {
         "Community API",
         "API over the nDelius DB used by HMPPS Digital team applications and services", "Java"
       ).apply {
+        APIDocs("https://community-api.test.delius.probation.hmpps.dsd.io/swagger-ui.html").addTo(this)
```

Comment:
I notice this is a test URL, where-as the other swagger docs are production.  
Also it's not accessible unless on the MoJ VPN - maybe not an issue for a dev, but just an observation about the availability of the swagger docs.

Reply:
I'll approve anyway as it is factually correct and what we use right now, but maybe highlights a few inconsistencies we can look at in future.

Reply:
👍 Yes, unfortunately, this is the one that's widely shared with everyone at the moment (as far as I can see).

I want to go through the whole lot and settle on pre-production or production URLs at some point next week. I'll make an issue out of this with a deadline

Reply:
Opened #64.